travis-ci.js

```diff
@@ -0,0 +1,262 @@
+const path = require("path");
+const webdriver = require("selenium-webdriver");
+const chrome = require("selenium-webdriver/chrome");
+const chromedriver = require("chromedriver");
+const until = webdriver.until;
+const By = webdriver.By;
+const options = new chrome.Options();
+options.addArguments("no-sandbox");
+const builder = new webdriver.Builder();
+builder.forBrowser("chrome");
+builder.setChromeOptions(options);
+const driver = builder.build();
+const csv = require("./node_modules/fast-csv");
+const fs = require("fs");
+const os = require("os");
+
+var countPasses = 0;
+var countFailures = 0;
+var countPending = 0;
+var flagPending = null;
+var csvTitle = null;
+var csvModule = null;
+var csvName = null;
+var csvPass = null;
+var csvFail = null;
+var csvNA = null;
+var csvExecution = "auto";
+var csvSuite = "tests";
+
+(async function() {
+    let baselinejson = JSON.parse(fs.readFileSync("./test/tools/CI/baseline/baseline.config.json"));
```

Comment:
indent format:
```
  .html, package.json, .yml: two blanks
  baseline.config.json, travis-ci.js: Tab
```
Shall we keep two blanks for baseline.config.json, travis-ci.js?